code/modules/admin/verbs/listahelps.dm

```diff
@@ -0,0 +1,307 @@
+/datum/adminticket
+	var/ID = "" //ID of the ticket, very important as its used to find adminhelps.
+	var/user = "" //The user of the ahelp.
+	var/uckey //The saved ckey of the adminheloing user.
+	var/admin = "N/A" //The handling admin? Like come on.
+	var/msg = "" //The adminhelp message.
+	var/resolved = "No" //Is it resolved? Its much easier to have a "Yes" or a "No", as you can directly concat it into strings making life that much easier.
+	var/permckey = "" //The perm ckey, never removed essentially.
+	var/permuser = "" //Same as above!
+	var/uID = "" //The UNIQUE id, made by putting part of the ckey and the ID together. Used internally in code.
+	var/active = "No" //Is the adminhelp active, eg admin responded? This is the same as above, it makes life easier.
+	var/logs = list() //The logs of the adminhelp.
```

Comment:
Please do not initialize lists directly on the object, or in `/new` please use the `LAZY_` list helpers when working with lists.

More about that can be found [here](https://github.com/HippieStation/HippieStation13/blob/master/code/__HELPERS/lists.dm#L459)